### PR TITLE
Add local releaser script

### PR DESCRIPTION
**Pull request by loosebazooka**
- local releaser script queries for the latest release on github
- downloads all assets and signs with sigstore and gpg
- generates a maven central release bundle

Signed-off-by: Appu Goundan <appu@google.com>

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -1,2 +1,2 @@
 group=dev.sigstore
-version=dev-SNAPSHOT
+version=SNAPSHOT
```
Comment (loosebazooka) [inline gradle.properties:2]: this is intentional. dev was too weird as you pointed out.

File: scripts/sign_and_bundle_release.sh [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+cd $RELEASE_DIR
+
+# cosign sign all the files
+echo "signing with cosign"
+for file in *; do
+  # skip intoto attestations, they are already signed
+  if [ $file == *.intoto.jsonl ] ; then
+    continue;
+  fi
+  COSIGN_EXPERIMENTAL=1 cosign sign-blob --yes $file --output-signature=$file.sig --output-certificate=$file.pem
```
Comment (patflynn) [inline scripts/sign_and_bundle_release.sh:36]: for now you should also include the bundle.

**Review by patflynn — CHANGES_REQUESTED** [review]

File: scripts/sign_and_bundle_release.sh (hunk 2 above)
Comment (loosebazooka) [reply scripts/sign_and_bundle_release.sh:36]: what bundle?

Comment (loosebazooka) [reply scripts/sign_and_bundle_release.sh:36]: oh the rekor bundle?

**Review by patflynn — APPROVED** [review]

**Pull request merged**